src/pybind/mgr/dashboard_v2/HACKING.rst

```diff
@@ -315,21 +315,21 @@ Now only authenticated users will be able to "ping" your controller.
 How to access the manager module instance from a controller?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Each controller class derived from ``BaseController``has a class property that
-points to the manager module global instance. The property is named ``mgr``.
-There is another class property called ``logger`` to easily add log messages.
+We provide the manager module instance as a global variable that can be
+imported in any module. We also provide a logger instance in the same way.
 
 Example::
 
   import cherrypy
+  from .. import logger, mgr_inst as mgr
```

Comment:
can we either use `from .. import mgr` or `from .. import mgr_inst`?

Reply:
I agree, I'll change it to just `mgr`